packages/@lwc/state/src/types.ts

```diff
@@ -52,5 +52,6 @@ export type DefineState = <
     computed: MakeComputed,
     update: MakeUpdate,
```

Comment:
can we do something to let consumers know that `update` is deprecated?

Reply:
since the package version is `0.x.y` i was thinking about removing the update functionality entirely (working on it in a different PR).

Reply:
created https://github.com/salesforce/lightning-labs/compare/main...jodarove:lightning-labs:jodarove/remove-update-proposal?expand=1 (contains also this changes) (or [this pr against this branch](https://github.com/jodarove/lightning-labs/pull/1) to show the isolated changes)